modelskill/plot.py

```diff
@@ -64,6 +66,10 @@
 # register_option("plot.scatter.table.show", False, validator=settings.is_bool)
 register_option("plot.scatter.legend.fontsize", 12, validator=settings.is_positive)
 
+# TODO: Auto-implement
+# still requires plt.rcParams.update(modelskill.settings.get_option('plot.rcParams'))
+register_option("plot.rcParams", {}, settings.is_dict)  # still have to
```

Comment:
Could you please explain the need for changing the rcParams? And is this actually used? If needed maybe you could use the context manager https://matplotlib.org/stable/api/matplotlib_configuration_api.html#matplotlib.rc_context ?

Reply:
The concept is to load rcParams styling as defined in the `modelskill\styles\simple.yml` files. E.g. in this case I have changed the mathtext to regular instead of italics. But it could also be to define other generic styles. I want it to be a permanent override associated with the .yml styles, so I'm not sure if the context manager is the appropriate use here?

At the moment it doesn't update the rcParams unless you run:

```python
plt.rcParams.update(modelskill.settings.get_option('plot.rcParams'))
```

Ideally this would be done when calling
```python
modelskill.settings.load_style()
```

Happy to remove this if you don't agree.

Reply:
Alright. I wasn't aware. Just keep it